perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/AuthzResolverBlImpl.java

```diff
@@ -88,6 +100,36 @@
 
 	private final static Set<String> extSourcesWithMultipleIdentifiers = BeansUtils.getCoreConfig().getExtSourcesMultipleIdentifiers();
 
+	/**
+	 * Create a list from the given objects and then check if the principal is authorized.
+	 *
+	 * @param sess perunSession which contains the principal.
+	 * @param policyDefinition of policy which contains authorization rules.
+	 * @param listOfObjects on which will be authorization provided.
+	 * @param objects on which will be authorization provided.
+	 * @return true if the principal has particular rights, false otherwise.
+	 * @throws PolicyNotExistsException when the given policyDefinition does not exist in the PerunPoliciesContainer.
+	 */
+	public static boolean authorized(PerunSession sess, String policyDefinition, List<? extends PerunBean> listOfObjects, PerunBean... objects) throws PolicyNotExistsException {
+		Utils.notNull(sess, "sess");
+		Utils.notNull(listOfObjects, "listOfObjects");
+		return authorized(sess, policyDefinition, ImmutableList.copyOf(Iterables.concat(listOfObjects, Lists.newArrayList(objects))));
```

Comment:
is there any particular reason to have Immutable copy of the original list?

Reply:
Method authorized() now takes only List<PerunBean>, so this logic was erased.